.github/workflows/publish-v2.yml

```diff
@@ -81,6 +81,7 @@ jobs:
         if: env.CHANNEL != 'cloud'
         uses: ./.github/workflows/update-readme-txt
         with:
+          README_TXT_PATH: $GITHUB_WORKSPACE/elementor/readme.txt
```

Comment:
When the new one-click action is merged, the v2 action won't be needed anymore, making this redundant.

Reply:
yes, but I want to keep publishV2 until we test the workflow in real release. We need some back up. After that I'll remove publishV2 and README_TXT_PATH.